pkg/eks/update_config_test.go

```diff
@@ -0,0 +1,166 @@
+package eks_test
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	awseks "github.com/aws/aws-sdk-go/service/eks"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/stretchr/testify/mock"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	. "github.com/weaveworks/eksctl/pkg/eks"
+	"github.com/weaveworks/eksctl/pkg/testutils/mockprovider"
+)
+
+var _ = Describe("EKS API wrapper", func() {
+	Describe("can update cluster configuration for logging", func() {
+		var (
+			ctl *ClusterProvider
+
+			cfg *api.ClusterConfig
+			err error
+
+			clusterLogging []*awseks.LogSetup
```

Comment:
Rename to `sentClusterLogging` or something that indicates that this is the one sent to the AWS API

Reply:
Will do.